### PR TITLE
#Issue62 min splits configurable

### PR DESCRIPTION
**Pull request by AyushWalekar**
#62 min splits configurable
Added min_splits to github.properties to be read in RestConfig.java to override default '4'.

### CODE REVIEW

File: trino-rest-github/src/main/java/pl/net/was/rest/github/GithubRest.java [hunk 1]
```diff
@@ -925,7 +925,9 @@ public GithubRest(RestConfig config)
     {
         requireNonNull(config, "config is null");
         GithubRest.token = config.getToken();
-
+        if(config.getMinSplits() > 0) {
+            GithubRest.minSplits = config.getMinSplits();
+        }
```
Comment (nineinchnick) [inline trino-rest-github/src/main/java/pl/net/was/rest/github/GithubRest.java:930]: If someone tries setting this to 0, it'll be 4 instead, which would be surprising. You can use `com.google.common.base.Verify.verify` here to ensure this value is not 0 and provide an actionable error message:
```suggestion
        verify(config.getMinSplits() > 0, "min_splits must be greater than zero");
        GithubRest.minSplits = config.getMinSplits();
```

File: trino-rest-base/src/main/java/pl/net/was/rest/RestConfig.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
     private String customerSecret;
     private String secret;
     private String token;
+    private int minSplits;
```
Comment (nineinchnick) [inline trino-rest-base/src/main/java/pl/net/was/rest/RestConfig.java:28]: Can you add a default value of 1 here?

File: trino-rest-github/src/main/java/pl/net/was/rest/github/GithubRest.java [hunk 3]
```diff
@@ -129,8 +129,8 @@
     public static final String SCHEMA_NAME = "default";
 
     private static final int PER_PAGE = 100;
-    private static final int MIN_SPLITS = 4;
 
+    private static int minSplits = 4;
```
Comment (nineinchnick) [inline trino-rest-github/src/main/java/pl/net/was/rest/github/GithubRest.java:133]: If you'd add a default value in the Config, this one is not needed anymore.

**Review by nineinchnick — APPROVED** [review]
Thanks a lot! You're the first contributor in this repo :-)

**Comment by AyushWalekar** [comment]
@nineinchnick Apologies for the delay, all my notifications went to spam for some reason. I see the changes are already done.
Thank you so much! 😃

**Pull request merged**